docs/sync.js

```diff
@@ -261,3 +261,104 @@ class Session {
 	 */
 	get state() {}
 }
+
+
+/**
+ * Class for creating custom Data Connectors. Only available in the Enterprise Edition.
+ * @memberof Realm.Sync
+ */
+class Adapter {
+	/**
+	 * Create a new Adapter to moitor and process changes made across multiple Realms
+	 * @param {string} local_path - the local path where realm files are stored
```

Comment:
Is this where the synced realms that are being monitored will be stored? Or is it for some metadata realm? Does this mean that my server running the adapter should potentially have storage available comparable to my actual database server (or, in case they're the same, twice the space necessary for the data)?